net/base/net_export.h

```diff
@@ -4,12 +4,13 @@
 
 #ifndef NET_BASE_NET_EXPORT_H_
 #define NET_BASE_NET_EXPORT_H_
-
+#include "starboard/configuration.h"
 // Defines NET_EXPORT so that functionality implemented by the net module can
 // be exported to consumers, and NET_EXPORT_PRIVATE that allows unit tests to
 // access features not intended to be used directly by real consumers.
 
-#if defined(COMPONENT_BUILD)
+// TODO b/296506338 Decide how to export symbols for base and net
```

Comment:
Tiny nit: your TODO format looks slightly off, i believe we landed on `TODO(b/296506338): ` as a canonical way to do those, but i don't think we are enforcing it anywhere.

Reply:
// TODO: b/87654321 - Migrate this code out to third_party.  - i got this format after looking at latest C++ google todo guidelines 

I can stick to what we have instead

Reply:
Can you point at those guidelines please ? If that's Google/Chromium adopted, we should follow.